pkg/virtctl/vm/vm.go

```diff
@@ -323,105 +97,6 @@ func usage(cmd string) string {
-  {{ProgramName}} addvolume fedora-dv --volume-name=example-dv --serial=1234567890
-
-  #Dynamically attach a volume to a running VM and persisting it in the VM spec. At next VM restart the volume will be attached like any other volume.
-  {{ProgramName}} addvolume fedora-dv --volume-name=example-dv --persist
-
-  #Dynamically attach a volume with 'none' cache attribute to a running VM.
-  {{ProgramName}} addvolume fedora-dv --volume-name=example-dv --cache=none
-  `
-}
-
-func usageRemoveVolume() string {
-	return `  #Remove volume that was dynamically attached to a running VM.
-  {{ProgramName}} removevolume fedora-dv --volume-name=example-dv
-
-  #Remove volume dynamically attached to a running VM and persisting it in the VM spec.
-  {{ProgramName}} removevolume fedora-dv --volume-name=example-dv --persist
-  `
-}
-
-func addVolume(vmiName, volumeName, namespace string, virtClient kubecli.KubevirtClient, dryRunOption *[]string) error {
-	volumeSource, err := getVolumeSourceFromVolume(volumeName, namespace, virtClient)
-	if err != nil {
-		return fmt.Errorf("error adding volume, %v", err)
-	}
-	hotplugRequest := &v1.AddVolumeOptions{
-		Name: volumeName,
-		Disk: &v1.Disk{
-			DiskDevice: v1.DiskDevice{
-				Disk: &v1.DiskTarget{
-					Bus: "scsi",
-				},
-			},
-		},
-		VolumeSource: volumeSource,
-		DryRun:       *dryRunOption,
-	}
-	if serial != "" {
-		hotplugRequest.Disk.Serial = serial
-	} else {
-		hotplugRequest.Disk.Serial = volumeName
-	}
-	if cache != "" {
-		hotplugRequest.Disk.Cache = v1.DriverCache(cache)
-		// Verify if cache mode is valid
-		if hotplugRequest.Disk.Cache != v1.CacheNone &&
-			hotplugRequest.Disk.Cache != v1.CacheWriteThrough &&
-			hotplugRequest.Disk.Cache != v1.CacheWriteBack {
-			return fmt.Errorf("error adding volume, invalid cache value %s", cache)
-		}
-	}
-	if !persist {
-		err = virtClient.VirtualMachineInstance(namespace).AddVolume(context.Background(), vmiName, hotplugRequest)
-	} else {
-		err = virtClient.VirtualMachine(namespace).AddVolume(vmiName, hotplugRequest)
-	}
-	if err != nil {
-		return fmt.Errorf("error adding volume, %v", err)
-	}
-	fmt.Printf("Successfully submitted add volume request to VM %s for volume %s\n", vmiName, volumeName)
-	return nil
-}
-
-func removeVolume(vmiName, volumeName, namespace string, virtClient kubecli.KubevirtClient, dryRunOption *[]string) error {
-	var err error
-	if !persist {
-		err = virtClient.VirtualMachineInstance(namespace).RemoveVolume(context.Background(), vmiName, &v1.RemoveVolumeOptions{
-			Name:   volumeName,
-			DryRun: *dryRunOption,
-		})
-	} else {
-		err = virtClient.VirtualMachine(namespace).RemoveVolume(vmiName, &v1.RemoveVolumeOptions{
-			Name:   volumeName,
-			DryRun: *dryRunOption,
-		})
-	}
-	if err != nil {
-		return fmt.Errorf("error removing volume, %v", err)
-	}
-	fmt.Printf("Successfully submitted remove volume request to VM %s for volume %s\n", vmiName, volumeName)
-	return nil
-}
-
 func gracePeriodIsSet(period int64) bool {
```

Comment:
this function can be probably moved to stop command file, because it is used only there

Reply:
Might change this to use `flags.Changed` instead of introducing a generic `-1` constant. And maybe this can be inlined as well.